src/main/kotlin/lotto/dto/InputLottoMachineRequestDto.kt

```diff
@@ -0,0 +1,35 @@
+package lotto.dto
+
+import lotto.domin.LottoNumberSet
+
+data class InputLottoMachineRequestDto(
+    val payment: Int,
+    val manualLottoCount: Int,
+    val manualLotto: List<LottoNumberSet>,
+) {
+
+    companion object {
+
+        const val CANNOT_CONVERT_INT = "정수로 변환할 수 없습니다."
+        const val NEGATIVE_ERROR = "음수는 입력할 수 없습니다."
+
+        fun of(inputPayment: String, inputManualLottoCount: String, manualLotto: List<LottoNumberSet>): InputLottoMachineRequestDto {
```

Comment:
```suggestion
        fun of(
            inputPayment: String,
            inputManualLottoCount: String,
            manualLotto: List<LottoNumberSet>
        ): InputLottoMachineRequestDto {
```
파라미터가 긴 경우에는 (2개 혹은 3개 이상) 개행해주시면 가독성이 좋아져요

Reply:
inputManualLottoCount는 manualLotto의 size로 대체할 수 있지 않을까요?